bootstrap/command/mocha

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+#
+# Copyright (c) 2017 The Lunch Class Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+. $BACARDI_PATH/bootstrap/common/path_info.sh
+. $BACARDI_PATH/bootstrap/common/platform_info.sh
+
+# TODO: should be set mocha path for windows
+$(node_modules_path)/.bin/mocha $@
```

Comment:
I think you can just use ```mocha $@``` because we already add .bin directory to env.path in bootstrap/bacardi.sh. (line number 27)

Reply:
i removed this file and i change to use `mocha $@` directly in bootstrap/command/test